ISSUE_TEMPLATE.md

```diff
@@ -22,18 +56,12 @@ import numpy; print("NumPy", numpy.__version__)
 import scipy; print("SciPy", scipy.__version__)
 import sklearn; print("Scikit-Learn", sklearn.__version__)
 ```
-- Please be specific about what estimators and/or functions are involved
-and the shape of the data, as appropriate; please include a
-[reproducible](http://stackoverflow.com/help/mcve) code snippet
-or link to a [gist](https://gist.github.com). If an exception is raised,
-please provide the traceback.
-- Please ensure all code snippets and error messages are formatted in
-appropriate code blocks.
-See ["Creating and highlighting code blocks"](https://help.github.com/articles/creating-and-highlighting-code-blocks).
-
-If you are submitting an algorithm or feature request:
-- Please verify that the algorithm fulfills our
-[new algorithm requirements](http://scikit-learn.org/stable/faq.html#can-i-add-this-new-algorithm-that-i-or-someone-else-just-published).
-
-Thanks for contributing! Please delete these guidelines before submitting
-your issue.
\ No newline at end of file
+-->
+- Platform:
```

Comment:
I would remove these lines, you just want people to paste the output of the snippet above.